src/lib/Browser.js

```diff
@@ -184,7 +184,7 @@ class Browser {
         if (!gl) {
             const canvas = document.createElement('canvas');
             // Should stop 'Rats! WebGL hit a snag' error when checking WebGL support
-            canvas.addEventListener(EVENT_WEBGL_CONEXT_LOST, (e) => {
+            canvas.addEventListener(EVENT_WEBGL_CONTEXT_LOST, (e) => {
```

Comment:
Can we remove the supportsWebGL global variable?

Reply:
Will still be useful if someone is viewing a folder that has multiple 3D files since that result will be cached between previews.